src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -135,6 +135,7 @@ export function generatePerformanceEventView(
   const searchQuery = decodeScalar(query.query) || '';
   const conditions = tokenizeSearch(searchQuery);
   conditions.setTagValues('event.type', ['transaction']);
+  conditions.setTagValues('transaction.duration', ['<15m']);
```

Comment:
Is this filter intended to filter out long running browser transactions? Is this in addition to the SDK side filtering we talked about or instead of it?

Reply:
Pretty sure this is just a ui side patch for the time being

Reply:
This is meant to be in addition to the SDK side filtering. If we find that the SDK is able to filter out enough of the outliers then we can come back and remove this after.